config/jobs/kubernetes/sig-k8s-infra/trusted/sig-release-release-team-jobs/release-team-periodics.yaml

```diff
@@ -2,7 +2,7 @@ periodics:
 - name: periodic-sync-enhancements-github-project-1-29
 # temporarily disable job by with an impossible cron date instead of deleting it
   cron: '0 0 31 2 *'
```

Comment:
Can you comment out the cron, and push it in a single commit, please?

Reply:
A single commit without the cron, or a single commit including both changes?

Reply:
both changes